mods/beds/functions.lua

```diff
@@ -5,6 +5,16 @@ if enable_respawn == nil then
 	enable_respawn = true
 end
 
+-- Physics override management mods (shadow the global variable)
+local player_monoids = core.get_modpath("player_monoids") and _G["player_monoids"]
```

Comment:
using `core.global_exists` would be nice and I wonder if we should be checking both the mod existence and global? just the latter would be enough.

Reply:
`local a = core.global_exists("a") and _G.a` sounds ok

Reply:
This does not check against global namespace poisoning, but that shouldn't happen anyway. At the end of the day there's many solutions and I don't think it really matters at all.